packages/react/src/internal/keyboard/navigation.js

```diff
@@ -68,5 +68,5 @@ export const selectorFocusable = `
   a[href], area[href], input:not([disabled]),
   button:not([disabled]),select:not([disabled]),
   textarea:not([disabled]),
-  iframe, object, embed, *[tabindex], *[contenteditable=true]
+  iframe, object, embed, *[tabindex]:not([disabled]), *[contenteditable=true]
```

Comment:
Would this work?

```suggestion
  iframe, object, embed, *[tabindex]:not(:disabled), *[contenteditable=true]
```

Reply:
yes that also works, I was following the convention from the previous selectors like `textarea:not([disabled])`. would one be preferred over the other here?

Reply:
Oh good point! `:disabled` doesn't work for custom elements, so probably we should keep this as-is.